micropython/senml/README.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+The KPN SenML library helps you create and parse [senml documents](https://tools.ietf.org/html/draft-ietf-core-senml-13)
```

Comment:
There shouldn't be any references to KPN in the code or docs.  Eg reword this as "The SenML library ...".

Reply:
What about the license header in `*.py` files ?

Reply:
I updated all the license headers.